package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-js",
-  "version": "9.26.1",
+  "version": "9.27.0",
```

Comment:
@TSLarson we need not update package version in a non-release PR.
We will do it as part of the release.
Can you please revert these and other related changes

You can refer the changelog and previous PRs mentioned in those changeLog.md
https://github.com/auth0/auth0.js/blob/master/CHANGELOG.md

Sample previous PR - https://github.com/auth0/auth0.js/pull/1433/files

Reply:
just removed this TY!